packages/client/src/pages/page-editor/page-editor.vue

```diff
@@ -78,8 +78,8 @@
 </MkSpacer>
 </template>
 
-<script lang="ts">
-import { defineComponent, defineAsyncComponent, computed } from 'vue';
+<script lang="ts" setup>
+import { defineAsyncComponent, provide, watch } from 'vue';
 import 'prismjs';
 import { highlight, languages } from 'prismjs/components/prism-core';
```

Comment:
Good catch of removing the unused `highlighter` function further down, but you should also remove these unused imports then.